compiler/rustc_metadata/src/creader.rs

```diff
@@ -751,7 +765,7 @@ impl<'a> CrateLoader<'a> {
         };
         info!("panic runtime not found -- loading {}", name);
 
-        let cnum = self.resolve_crate(name, DUMMY_SP, CrateDepKind::Implicit);
+        let cnum = self.resolve_crate_or_abort(name, DUMMY_SP, CrateDepKind::Implicit);
```

Comment:
It should be fine to just return on `None` here and in `fn inject_profiler_runtime`, thus eliminating the need in `fn resolve_crate_or_abort`.